src/utils/middleware.js

```diff
@@ -1,9 +1,43 @@
+import Swagger from '@aeternity/aepp-sdk/es/utils/swagger';
+import { camelCase } from 'lodash-es';
 import { wrapTry } from './index';
 
-const backendFetch = (path, ...args) => wrapTry(
+const middlewareFetch = (path, ...args) => wrapTry(
```

Comment:
I don't think that `wrapTry` wrapper is needed here, as we don't do that for requesting a swagger file of the node.